src/layouts/LandingLayout/index.tsx

```diff
@@ -60,14 +60,15 @@ export default function LandingLayout({ children }: Readonly<PropsWithChildren>)
           </Grid>
         </section>
         <section className="section">
-          <Grid container spacing={0}>
+          <Grid container spacing={0} maxWidth={1280} margin="auto" alignItems="center">
```

Comment:
Here, instead of hardcoding the maxWidth, it'll be better to do:
```
sx={(theme) => ({ maxWidth: theme.breakpoints.values.lg })}
```

That way, if that brakpoint changes, everything else will too and it'll be easier to mantain

Reply:
Hi, ok, I'm going to add it